### PR TITLE
core: Fix typos (found by codespell)

### PR DESCRIPTION
**Pull request by stweil**
Signed-off-by: Stefan Weil sw@weilnetz.de

### CODE REVIEW

**Comment by stweil** [comment]
@owncloud-bot: My contribution is MIT licensed.

File: core/l10n/ast.json [hunk 1]
```diff
@@ -116,7 +116,7 @@
     "change" : "camudar",
     "delete" : "desaniciar",
     "access control" : "control d'accesu",
-    "An error occured. Please try again" : "Hebo un fallu. Por favor, inténtalo dempués. ",
```
Comment (DeepDiver1975) [inline core/l10n/ast.json]: no manual changes to files within the l10n folders - these will be recreated and overwritten on the next transifex sync run

**Comment by stweil** [comment]
@DeepDiver1975, are you sure that you only want the corresponding source changes (which are also included)? Then code and translation no longer match, and there won't be a translation for the fixed strings.

Ideally the next transifex sync run won't overwrite anything because it creates the same result as my patch.

If you insist, I can remove the l10n changes (with the consequence of missing translations).

**Comment by DeepDiver1975** [comment]
> Ideally the next transifex sync run won't overwrite anything because it creates the same result as my patch.
> 
> If you insist, I can remove the l10n changes (with the consequence of missing translations).

on the next sync run all translation will be wiped because they are not yet on transifex .. they will flow in in the next days then

**Comment by stweil** [comment]
I updated the pull request without the patches for `core/l10n` now.

**Comment by LukasReschke** [comment]
:+1:

File: core/js/tests/lib/sinon-1.15.4.js [hunk 2]
```diff
@@ -201,7 +201,7 @@
         // contain already visited objects
         var objects1 = [],
             objects2 = [],
-        // contain pathes (position in the object structure)
+        // contain paths (position in the object structure)
```
Comment (MorrisJobke) [inline core/js/tests/lib/sinon-1.15.4.js]: Changes in this file are useless, because this is a 3rdparty library and should be fixed upstream.

**Comment by stweil** [comment]
Updated PR, removed `core/js/tests/lib/sinon-1.15.4.js` from the patch as suggested by @MorrisJobke and rebased to latest git master.

**Comment by MorrisJobke** [comment]
Looks nice :+1:

**Pull request merged**